src/backend/base/langflow/graph/vertex/base.py

```diff
@@ -13,6 +13,7 @@
 import pandas as pd
 from loguru import logger
 
+from langflow.custom import Component
```

Comment:
Expected an Error here from Mypy/ lint since Component was used only for TypeChecking.
But works without error.
@cbornet do we have an exception added for this scenario?